src/test/java/ru/netology/manager/AfishaManagerTest.java

```diff
@@ -0,0 +1,110 @@
+package ru.netology.manager;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import ru.netology.domain.Movie;
+import static org.junit.jupiter.api.Assertions.*;
+
+class AfishaManagerTestNoEmpty {
+    private AfishaManager manager = new AfishaManager();
+    Movie first = new Movie(1, "img1.jpg", "Name 1", "Comedy" );
+    Movie second = new Movie(2, "img1.jpg", "Name 2", "Drama" );
+    Movie third = new Movie(3, "img1.jpg", "Name 3", "Documentary" );
+    Movie fourth = new Movie(4, "img1.jpg", "Name 4", "Fantastic" );
+    Movie fifth = new Movie(5, "img1.jpg", "Name 5", "Detective" );
+    Movie sixth = new Movie(6, "img1.jpg", "Name 6", "Cartoon" );
+    Movie seventh = new Movie(7, "img1.jpg", "Name 7", "Anime" );
+    Movie eighth = new Movie(8, "img1.jpg", "Name 8", "Romantic" );
+    Movie ninth = new Movie(9, "img1.jpg", "Name 9", "Horror" );
+    Movie tenth = new Movie(10, "img1.jpg", "Name 10", "Historical" );
+    Movie eleventh = new Movie(11, "img1.jpg", "Name 11", "Comedy" );
+
+
+    @BeforeEach
+    public void setUp(){
+        manager.add(first);
+        manager.add(second);
+        manager.add(third);
+        manager.add(fourth);
+        manager.add(fifth);
+        manager.add(sixth);
+        manager.add(seventh);
+        manager.add(eighth);
+        manager.add(ninth);
+    }
+
+    @Test
+    void ShouldShowWhenZero() {
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.